src/uattributes/uattributesvalidator.rs

```diff
@@ -590,17 +593,20 @@ mod tests {
             ttl,
             ..Default::default()
         };
+
+        thread::sleep(std::time::Duration::from_millis(created_millis_ago));
+
```

Comment:
FMPOV it is an anti-pattern to rely on sleeping in test code. Please replace this with the original or a clock-tick based approach.

Reply:
OK -- I think I was able to pull this off. Can you take another look @sophokles73?